### PR TITLE
Add buses logos

### LINKED ISSUE #32: add busses logos
[bus logos.zip](https://github.com/hasadna/open-bus-map-search/files/12519707/bus.logos.zip)
replace the icons on the map to better icons
image

לכל נקודה על המפה יש attribute בשם operator_id. אגד למשל עם id=3 לכן התמונה היא 3.svg

### PR DESCRIPTION
**Pull request by rzarviv**
- added buses and operator's logos to `public` directory.
- show buses logos according to their operator_id in real-time map

close #32

### CODE REVIEW

File: src/pages/Map.scss [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 .mask {
     display: block;
-    mask-repeat: no-repeat;
-    mask-position: center;
-    mask-size: contain;
-    -webkit-mask-repeat: no-repeat;
+    /*-webkit-mask-repeat: no-repeat;
     -webkit-mask-position: center;
-    -webkit-mask-size: contain;
+    -webkit-mask-size: contain;*/
```
Comment (NoamGaash) [inline src/pages/Map.scss:15]: remove comments where possible

**Review by NoamGaash — APPROVED** [review]
The code looks great!
I'll test it manually and then add some screenshots to the PR before merging it
Good job :+1: :100: :1st_place_medal:

**Comment by NoamGaash** [comment]
@rzarviv please approve my latest commit to your PR

**Comment by rzarviv** [comment]
@NoamGaash apparently I cannot approve your latest commit since I'm the creator of the PR, so I'll add a whitespace somewhere just to push it from my branch

**Comment by NoamGaash** [comment]
@rzarviv I meant I want your opinion,
if you believe it's good I'll merge it once the tests are passing

**Pull request merged**